bundle/Templating/Twig/BetterIbexaAdminUIExtension.php

```diff
@@ -18,4 +19,14 @@ public function getFunctions(): array
             ),
         ];
     }
+
+    public function getFilters()
+    {
+        return [
+            new TwigFunction(
+                'show_language_name',
```

Comment:
This should be `ngadmin_language` so we can fetch the object instead of the name, which allows some more flexibility.